.gitmodules

```diff
@@ -26,6 +26,6 @@
 [submodule "node-red-contrib-agile-solid"]
 	path = node-red-contrib-agile-solid
 	url = https://github.com/Agile-IoT/node-red-contrib-agile-solid.git
-[submodule "node-red-contrib-thingspeak42"]
-	path = node-red-contrib-thingspeak42
-	url = https://github.com/clough42/node-red-contrib-thingspeak42.git
+[submodule "node-red-contrib-agile-thingspeak"]
```

Comment:
Why this change? We don't have this repository yet. I don't know if @Exulansis is planning to fork the thingspeak42 node, but let's change it when forked.